### PR TITLE
MAGECLOUD-3968: Add static and unit tests

### PR DESCRIPTION
**Pull request by shiftedreality**
### Fixed Issues (if relevant)

1. https://magento2.atlassian.net/browse/MAGECLOUD-3968

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: phpcs-ruleset.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <description>Custom Magento ECE Tools coding standard.</description>
+
+    <rule ref="PSR2"/>
+
+    <rule ref="Generic.Functions.CallTimePassByReference"/>
+    <rule ref="Generic.PHP.DeprecatedFunctions"/>
+    <rule ref="Generic.Files.LineLength">
+        <properties>
+            <property name="lineLimit" value="120"/>
+            <property name="absoluteLineLimit" value="120"/>
```
Comment (bbatsche) [inline phpcs-ruleset.xml]: Just pointing out, PSR-2 explicitly says

> The soft limit on line length MUST be 120 characters; automated style checkers MUST warn but MUST NOT error at the soft limit.

But hey, oh well.

**Review by bbatsche — APPROVED** [review]
Are you planning to add PHPStan and/or PHPMD in a later PR?

**Comment by shiftedreality** [comment]
Yes, more tests will be added later

File: tests/static/Sniffs/Directives/StrictTypesSniff.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
```
Comment (arhiopterecs) [inline tests/static/Sniffs/Directives/StrictTypesSniff.php]: Excess line

File: tests/static/Sniffs/Whitespace/MultipleEmptyLinesSniff.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
```
Comment (arhiopterecs) [inline tests/static/Sniffs/Whitespace/MultipleEmptyLinesSniff.php:5]: Missing `declare(strict_types=1);`

Comment (shiftedreality) [reply tests/static/Sniffs/Whitespace/MultipleEmptyLinesSniff.php:5]: Added

File: tests/static/Sniffs/Directives/StrictTypesSniff.php (hunk 2 above)
Comment (shiftedreality) [reply tests/static/Sniffs/Directives/StrictTypesSniff.php]: Fixed

**Review by arhiopterecs — APPROVED** [review]

**Pull request merged**